packages/api-client-core/src/GadgetRecord.ts

```diff
@@ -9,66 +9,75 @@ export enum ChangeTracking {
 
 export type RecordShape = Record<string, any> | null | undefined | void;
 
+const kFields = Symbol.for("g/fields");
+const kInstantiatedFields = Symbol.for("g/if");
+const kPersistedFields = Symbol.for("g/pf");
+const kFieldKeys = Symbol.for("g/fk");
+const kTouched = Symbol.for("g/t");
```

Comment:
nit: Any reason to not go for clearer names?

Reply:
Yeah, bundle size -- these strings don't get mangled during minification. EVERY BYTE COUNTS